test/extended/cli/explain.go

```diff
@@ -425,6 +418,23 @@ var (
 		},
 	}
 
+	// This list holds apigroups whose resources are not similar for OpenShift and MicroShift.
```

Comment:
It's not really clear from the comments how the items in this list are different from the items in the builtinExceptionTypes list.

Reply:
The reasoning is same for both the list. 

As `oc explain` doc doesn't contain `spec+status` for a few apigroups, they were listed separately in `specialTypes` list, and tested against a special `field` and `pattern`. 

`specialExceptionTypes` is just taken out from `specialTypes` list due to same reason as `builtinExceptionTypes` 

If there is any suggestion, pls let me know. I'll update it.